src/diffusers/loaders.py

```diff
@@ -852,9 +936,6 @@ def load_lora_weights(self, pretrained_model_name_or_path_or_dict: Union[str, Di
         weight_name = kwargs.pop("weight_name", None)
         use_safetensors = kwargs.pop("use_safetensors", None)
 
-        # set lora scale to a reasonable default
-        self._lora_scale = 1.0
-
```

Comment:
This is the default value if it isn't set, no need to mutate the instance

Reply:
Why this change? I don't understand how this is needed to make LoRA compatible with T5? `load_lora_weights` doesn't return anything so I see no advantage to making this a class method. In order to call this method, the pipeline has to be instantiated IMO